### DISCUSSION TITLE
Keymap files

### DISCUSSION THREAD

**Discussion by botakkeren** [barry-ran/QtScrcpy#462] [Q&A]
can someone upload an PUBG M and CODM keymap file?? thanks

**Comment by dijo7** [comment]
[PUBG.txt](https://github.com/barry-ran/QtScrcpy/files/6908321/PUBG.txt)
Save as .json file

Use ATNSoft Key Remapper:
Assign arrow key up as scroll wheel up for primary and down for secondary weapon selection

Note: a custom layout settings based on BGMI
Display settings: 1980x1080 (try GFX Tool)
Smallest width: 411dp

BGMI in-game layout:
Controls type: Left(fixed button) Right(fixed Button)
Cloud saved map layout  code:   6974-6395-1015-9100-101

Feel free to edit the positions, its not default settings and sensitivity based on your mobile screen aspect ratio.

**Reply by pepijawa** [reply]
hello how to change keymap with antsoft?

**Reply by SabunMndi** [reply]
how you make the json file?

**Reply by JoyalZixx** [reply]
just rename it as json

**Reply by JoyalZixx** [reply]
this is for bgmi not for pubg, so the code wont work, burhhhhhhhhhhhh

**Reply by LalatinaBD** [reply]
@dijo7 do you have any idea how to turn on switchmap while using kmt_clicl_multi. even if i put switchmap : true, the mouse does not work.

**Comment by Speeductor** [comment]
Qtscrcpy keymap for pubg mobile, 2-3fingers control available, hola me

**Comment by mehran5609** [comment]
if anyone using qtscrcpy keymapping for the free fire max 
pls upload here
